model/src/main/java/com/basistech/rosette/apimodel/AdmRequest.java

```diff
@@ -48,12 +49,25 @@
      */
     private final LanguageCode language;
 
+
+    public AdmRequest(String profileId,
```

Comment:
Should this have the `@Builder` annotation like the one below?

Reply:
Having 2 Builder annotations might lead to confusion. Also the IDE only process the very 1st constructor with `@Builder`, and yields false-positive of compiler error when the caller uses the `genre` field.